hyper/http20/connection.py

```diff
@@ -51,8 +51,10 @@ class HTTP20Connection(object):
     :param enable_push: (optional) Whether the server is allowed to push
         resources to the client (see
         :meth:`get_pushes() <hyper.HTTP20Connection.get_pushes>`).
+    :SSLContext: (optional) A class with custom certificate settings. If not provided
+        then hyper's default SSLContext is used instead.
     """
-    def __init__(self, host, port=None, window_manager=None, enable_push=False,
+    def __init__(self, host, port=None, window_manager=None, enable_push=False, SSLContext=None,
```

Comment:
Can we keep the name of this parameter in keeping with the style of the rest? `ssl_context` is preferred. =)